amazon-kinesis-client/src/test/java/software/amazon/kinesis/application/TestConsumer.java

```diff
@@ -116,6 +126,13 @@ public void run() throws Exception {
         }
     }
 
+    private void verifyCrossAccountCreds() {
+        if (consumerConfig.getCrossAccountCredentialsProvider() == null) {
+            throw new RuntimeException("To run cross account integration tests, pass in an AWS profile with -D" +
+                    KCLAppConfig.CROSS_ACCOUNT_PROFILE_PROPERTY);
+        }
+    }
```

Comment:
i think ideally the credentials providers should be abstracted away from this class
i left a comment about potentially making the `getCrossAccountCredentialsProvider` method `protected` and renaming it to `getCredentialsProviderForStreamOwner`
we can likely remove this logic in that approach if you find it suitable